include/rangers-api/Application/Foundation/AppHeapManager.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+namespace app::fnd {
+    class AppHeapManager {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.